tracing-log/tests/log_tracer.rs

```diff
@@ -70,8 +76,8 @@ fn normalized_metadata() {
                 target: "log_tracer".to_string(),
                 level: Level::INFO,
                 module_path: Some("log_tracer".to_string()),
-                file: Some("tracing-log/tests/log_tracer.rs".to_string()),
-                line: Some(64),
+                file: Some(my_file.to_string()),
+                line: Some(70),
```

Comment:
I think it would be better if we could avoid having to hardcode the line number here and a little further down in the file. This means that if we add any more code above this in the file, this test will break, even though the _behavior_ being tested is still correct.

What do you think about using [`log::Record::builder`](https://docs.rs/log/0.4.8/log/struct.Record.html#method.builder) instead of using the `info!` macro? That way, we could specify fixed values for the file and line number, and expect them to stay the same even if the test code changes and the actual line number is different?

Reply:
I just started with the idea of testing the "real" code people could write, but the `info` macro layer is definitely not useful here!

Reply:
Yeah, that's reasonable. IMO, though, the main thing we're testing here is whether the metadata is normalized correctly, so it's not really necessary to test the whole thing end to end if that also introduces more places where it could fail spuriously.